diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter2/kdoc/KdocFormatting.kt

```diff
@@ -219,7 +219,6 @@ class KdocFormatting(configRules: List<RulesConfig>) : DiktatRule(
                     .first()
                     .node
                     .startOffset, basicTags.first().node) {
-                val kdocSection = node.getFirstChildWithType(KDOC_SECTION)!!
```

Comment:
I still do not understand why this caused an issue - why this node did not have ChildWithType(KDOC_SECTION)

Reply:
there are 2 KDOC_SECTION s in this kdoc
1. with description
2. with tags
we get the first one and try to replace tag @property which is in another section.
assert check that anchor should be the child of current section.
so it is better and more universally just to get parent of the replacing tag

Reply:
ah, got it thanks!